### ISSUE TITLE
TypeError: 'instancemethod' object has no attribute '__getitem__'

### ISSUE THREAD

**Issue by PygmalionChen** [MLDroid/graph2vec_tf#2]
prev_iter_node_label = get_int_node_label(g.nodes[node]['relabel'][prev_iter]) is sth wrong.

**Comment by annamalai-nr** [comment]
There is nothing wrong here. Could you please more specific with the question? Thanks.

**Comment by srinivas212** [comment]
I'm running into similar issue.

Environment:
Python 2.7.13 :: Anaconda 4.4.0 (64-bit)
TF 1.4.1 (also tried TF 1.5)  

python main.py --corpus ../data/kdd_datasets/proteins --class_labels_file_name ../data/kdd_datasets/proteins.Labels --batch_size 16 --embedding_size 128 --num_negsample 5
INFO:root:Loaded 1113 graph file names form ../data/kdd_datasets/proteins
loaded all graphs in 17.72 sec
initial relabeling done in 3.24 sec
Traceback (most recent call last):
  File "main.py", line 98, in <module>
    main(args)
  File "main.py", line 43, in main
    wlk_relabel_and_dump_memory_version(graph_files, max_h=wlk_h, node_label_attr_name=label_filed_name)
  File "/mnt/raid10-storage/ssridha2/deep/tf/apps/graph2vec_tf/src/make_graph2vec_corpus.py", line 152, in wlk_relabel_and_dump_memory_version
    graphs = [wl_relabel(g, it) for g in graphs]
  File "/mnt/raid10-storage/ssridha2/deep/tf/apps/graph2vec_tf/src/make_graph2vec_corpus.py", line 62, in wl_relabel
    prev_iter_node_label = get_int_node_label(g.nodes[node]['relabel'][prev_iter])
TypeError: 'instancemethod' object has no attribute '__getitem__'

**Comment by kwonoh** [comment]
I got the same error. 
```
> python2 main.py --corpus ../data/kdd_datasets/mutag --class_labels_file_name ../data/kdd_datasets/mutag.Labels 
INFO:root:Loaded 188 graph file names form ../data/kdd_datasets/mutag
loaded all graphs in 0.15 sec
initial relabeling done in 0.02 sec
Traceback (most recent call last):
  File "main.py", line 98, in <module>
    main(args)
  File "main.py", line 43, in main
    wlk_relabel_and_dump_memory_version(graph_files, max_h=wlk_h, node_label_attr_name=label_filed_name)
  File "/Users/kwonoh/opt/repo/graph2vec_tf/src/make_graph2vec_corpus.py", line 152, in wlk_relabel_and_dump_memory_version
    graphs = [wl_relabel(g, it) for g in graphs]
  File "/Users/kwonoh/opt/repo/graph2vec_tf/src/make_graph2vec_corpus.py", line 62, in wl_relabel
    prev_iter_node_label = get_int_node_label(g.nodes[node]['relabel'][prev_iter])
TypeError: 'instancemethod' object has no attribute '__getitem__'
```
@MLDroid  Could you please let us know the exact version of networkx you used?

Thank you,
Oh-Hyun

**Comment by annamalai-nr** [comment]
Pls see README file for the library versions. Thanks.

PS:
tensorflow (version == 1.4.0)
networkx (version <= 2.0)

**Comment by kwonoh** [comment]
Thanks for the reply.
I am using networkx 1.11 but still have the issue (this is why I wondered the exact version you are using).

**Comment by kwonoh** [comment]
The issue is fixed when I use the networkx 2.1 instead of 1.11.

There are major changes in networkx between 1.11 and 2.0. 
It would be much clear if the dependency information in README is `networkx 2.0 or higher` or just `networkx==2.0` (not `networkx <= 2.0`)
